Tools/Taxonomy/Cwe/CweTaxonomyGenerator.cs

```diff
@@ -0,0 +1,331 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CsvHelper;
+using Microsoft.CodeAnalysis.Sarif;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Serialization;
+using Taxonomy.Common;
+
+namespace Taxonomy.Cwe
+{
+    public class CweTaxonomyGenerator : TaxonomyGenerator
+    {
+        public List<CweCsvRecord> ReadFromCsv(string filePath)
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

in theory, all methods  besides SaveToCsvToSarif could be private

Reply:
yes, I have changed all generators.